net/snort3/Makefile

```diff
@@ -27,8 +28,9 @@ define Package/snort3
   SECTION:=net
   CATEGORY:=Network
   DEPENDS:= \
+	    +@!(TARGET_powerpc||TARGET_mips):gperftools-runtime \
```

Comment:
While looking at https://github.com/openwrt/packages/blob/89fa8033b851db004182c5e33b0728bc2f95b807/libs/gperftools/Makefile#L37 are you sure that powerpc and mips are sufficient? I think there should be included mips64 and most likely mipsel. 

What I gave you was just suggestion in my comment https://github.com/openwrt/packages/pull/24055#issuecomment-3075777965 and if we want to follow Linux kernel guidelines as OpenWrt in the main follows it, there should be: ``Suggested-by``, ``Reported-by``, etc.

Reply:
No, you are correct. I will modify, thank you!